src/app/loadout-drawer/LoadoutPopup.tsx

```diff
@@ -113,6 +114,9 @@ function LoadoutPopup({
   // For the most part we don't need to memoize this - this menu is destroyed when closed
 
   const numPostmasterItemsTotal = totalPostmasterItems(dimStore);
+  const language = useSelector(languageSelector);
+  const [loadoutQuery, setLoadoutQuery] = useState('');
+  const searchRegexp = startWordRegexp(loadoutQuery, language);
```

Comment:
i would prefer `escapeRegExp(s)/i` here, people use all sorts of wacky naming schemes and `rocket` won't find `fusion_rocket` and typing `3` won't show me all names with `3` in them like `omni3`
i am not positive it buys an improvement that's worth the slightly unexpected behavior

Reply:
Fair, I was just copying some of our other searches. Personally I'm not super into the start word stuff either.